web/js/components/image-download/lat-long-inputs.js

```diff
@@ -27,9 +24,12 @@ function Input({
   const update = () => {
     try {
       const newInputValue = Number(inputValue);
-      const newArray = lodashClone(boundingBoxArray);
+      const newArray = structuredClone(boundingBoxArray);
       newArray[index] = newInputValue;
-      const crsCorrectedExtent = olProj.transformExtent(newArray, CRS.GEOGRAPHIC, crs);
+      // Due to a bug (ol v7.4.0) in transformExtent, we need to create a polygon before transforming it.
```

Comment:
Is this comment accurate & necessary to check into the repo?

Reply:
I created a [issue](https://github.com/openlayers/openlayers/issues/14961) in OpenLayers for the behavior that was causing the bug. I figured that the comment would be a nice breadcrumb as to why we're not using `transformExtent` so that we don't accidentally "fix" it in the future until the bug is truly resolved.

Reply:
If I am parsing the response to that ticket correctly, they are saying not a bug in `transformExtent`...? 

I don't want to get too sidetracked on this, just want to make sure that our comments are not misleading.

Reply:
Yeah, `transformExtent` returns `[Infinity, Infinity, -Infinity, -Infinity]` when trying to transform extents around Antarctica. From my investigation, I believe that this is an issue with OpenLayers since the behavior is inconsistent in regards to which extents succeed and which ones fail. The `Infinity` extent then trips our `!isEmpty(crsCorrectedExtent)` check on line 36 and causes the described bug.

Reply:
I hadn't seen the response to my issue. I'll remove the comment accordingly.